1-js/01-getting-started/1-intro/article.md

```diff
@@ -100,15 +100,15 @@ While planning to learn a new technology, it's beneficial to check its perspecti
 
 The syntax of JavaScript does not suit everyone's needs. Different people want different features.
 
-That's normal, because projects and requirements are different for everyone.
+That's to be expected, because projects and requirements are different for everyone.
 
 So recently a plethora of new languages appeared, which are *transpiled* (converted) to JavaScript before they run in the browser.
 
-The modern tools make the transpilation very fast and transparent, actually allowing developers to code in another language, autoconverting it "under the hood".
+Modern tools make the transpilation very fast and transparent, actually allowing developers to code in another language and autoconverting it "under the hood".
 
 Examples of such languages:
 
-- [CoffeeScript](http://coffeescript.org/) is a "syntax sugar" for JavaScript, it introduces shorter syntax, allowing to write more precise and clear code. Usually Ruby guys like it.
```

Comment:
The gendered language here (i.e. "guys") seems really clunky and old fashioned. This could be changed to "Ruby devs", "Rubyists" etc. I'm also noticing many places in the tutorial where, for example, the user is referred to as "he".